ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -351,7 +351,7 @@ public enum DeploymentType {
       "Config to enable pull queries that scan over the data";
   public static final boolean KSQL_QUERY_PULL_TABLE_SCAN_ENABLED_DEFAULT = true;
 
-  public static final String KSQL_QUERY_STREAM_PULL_QUERY_ENABLED 
+  public static final String KSQL_QUERY_STREAM_PULL_QUERY_ENABLED
```

Comment:
nit: unrelated whitespace change to this PR

Reply:
It makes the whitespace consistent. Agreed it's unrelated to the PR, but it's a minor improvement, so should be fine.